channels/rail/client/rail_main.c

```diff
@@ -620,9 +620,11 @@ static void* rail_virtual_channel_client_thread(void* arg)
 
 			if ((error = rail_order_recv(rail, data)))
 			{
+				Stream_Free(data, TRUE);
 				WLog_ERR(TAG, "rail_order_recv failed with error %"PRIu32"!", error);
 				break;
 			}
+			Stream_Free(data, TRUE);
```

Comment:
Thank you for  the suggestion, it's good to me, I will refactor to avoid dulicating Stream_Free.